unit_tests/.vscode/settings.json

```diff
@@ -0,0 +1,55 @@
+{
```

Comment:
please be careful about what you `git add` - I'm also not sure how this got past our `gitignore`